### DISCUSSION TITLE
Business request

### DISCUSSION THREAD

**Discussion by 100millionusd** [filecoin-project/community#201] [General]
Good Morning, 
we are running a face recognition company. A client’s wish is that this data has to be stored decentralised and always be "burned" automatically after 14 days. 
Questions:
Is this possible with Filecoin?
Thank you. Paul

**Comment by jimpick** [comment]
Minimum storage duration on Filecoin is 6 months.

You might be able to encrypt the data, and then use DRM style techniques to restrict access - eg. secure enclaves, etc. DRM is mostly for securing access to things like movies, so it might not have the security properties you desire.

**Reply by 100millionusd** [reply]
Hi, thanks for the Answer. Storage duration for 6 months minimum... Does that mean that any file that is uploaded can only be deleted after 6 months?

**Reply by jimpick** [reply]
Files don't get deleted, but sectors expire. A miner might store a 6 month deal in a sector that lives for 18 months, for example.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
in general:

there is no way to make sure that data is actually deleted. if you need to "delete" data the only way would be to make inaccessible, eg. encrypt it and "delete' the key for it on your side
